scico/flax/train/traversals.py

```diff
@@ -37,7 +37,7 @@ def clip_positive(params: PyTree, traversal: ModelParamTraversal, minval: float
         minval: Minimum value to clip selected model parameters and keep
             them in a positive range. Default: 1e-4.
     """
-    params_out = traversal.update(lambda x: jnp.clip(x, a_min=minval), params)
```

Comment:
fine, could also `jnp.clip(x, min=minval)`

Reply:
Unfortunately whether that works is dependent on the jax version in use.